src/Passes/Source/Commandline/ConfigurationBindInterface.hpp

```diff
@@ -0,0 +1,92 @@
+#pragma once
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "Commandline/ParameterParser.hpp"
+
+#include "Llvm/Llvm.hpp"
+
+#include <iomanip>
+#include <iostream>
+#include <sstream>
+#include <type_traits>
+#include <typeindex>
+
+namespace microsoft
+{
+namespace quantum
+{
+
+    /// Interface class to bind a variable to a configuration flag. This class provides
+    /// the neccessary means to
+    class IConfigBind
```

Comment:
The file defines the class `IConfigBind` but the file is named `ConfigurationBindInterface.hpp`. Consider renaming the file to `IConfigBind.hpp`.

Reply:
Done.